internal/appconfig/machines.go

```diff
@@ -62,6 +62,53 @@ func (c *Config) ToReleaseMachineConfig() (*fly.MachineConfig, error) {
 	return mConfig, nil
 }
 
+func (c *Config) ToTestMachineConfig(machineCommand, machineImage, machineEntrypoint, machineIP string) (*fly.MachineConfig, error) {
+	// for whatever reason, we need to split the entry point, but not the command
+	entrypoint, err := shlex.Split(machineEntrypoint)
+	if err != nil {
+		return nil, err
+	}
+	command := []string{machineCommand}
```

Comment:
Overriding the entrypoint has to be optional, if not set, it has to use the image's default entrypoint.

In fact, shell splitting machineCommand is a must, otherwise this is inconsistent and only works because the test is overriding the image's entrypoint by `/bin/sh -c`

Reply:
Done!